app/assets/javascripts/admin/addon/components/simple-list.hbs

```diff
@@ -33,20 +33,33 @@
 {{/if}}
 
 <div class="simple-list-input">
-  <Input
-    @type="text"
-    @value={{this.newValue}}
-    placeholder={{i18n "admin.site_settings.simple_list.add_item"}}
-    class="add-value-input"
-    autocomplete="off"
-    autocorrect="off"
-    autocapitalize="off"
-  />
-
-  <DButton
-    @action={{fn this.addValue this.newValue}}
-    @disabled={{this.inputEmpty}}
-    @icon="plus"
-    class="add-value-btn btn-small"
-  />
+  {{#if this.isPredefinedList}}
+    {{#if (gt this.validValues.length 0)}}
+      <ComboBox
```

Comment:
I am pretty sure it's fine, but just double checking -- if someone had added a wrong/misspelled choice before we had this list, it would be fine right? Since it's just showing in the list of values?

Reply:
Yes, you are right. We changed only list with allow_any: false. In that case when someone typed wrong value, validation is displayed and that value is not saved

Reply:
Ok cool thanks for clarifying :)